### ISSUE TITLE
Request for Support / Sponsorship

### ISSUE THREAD

**Issue by FlorianRappl** [AngleSharp/AngleSharp#1163]
Over the years this project had great contributors and sponsors. Moving forward the last year has shown that dedicated support (e.g., as provided by AWS and JetBrains) is crucial to allocate time for maintenance and move it forward. I'd like to continue in this mode; not only sometimes cutting out some of my spare time, but actually being able to have dedicated time slots.

So with this sticky note I call to support this project. It would be really wonderful and there are still some plans for a potential v2 that would benefit a lot from additional support (as well as its ecosystem, esp. CSS / JS / ...).

### Background

In the past we already had some great sponsors who brought this project forward.

By far the largest contribution came from AWS:

<img src="https://github.com/AngleSharp/AngleSharp/assets/1766191/df38e39e-7803-4020-835f-5b979eb4c8d3" width="150">

Closely behind (and still active) is the sponsorship from JetBrains:

<img src="https://github.com/AngleSharp/AngleSharp/assets/1766191/f74a1fe8-805b-4088-92bf-c0b528785b06" width="150">

And other companies:

<a href="https://www.deskdirector.com/"><img src="https://avatars.githubusercontent.com/u/14305729?s=100&v=4"></a> | <a href="https://www.sparkdevnetwork.org/"><img src="https://avatars.githubusercontent.com/u/548191?s=100&v=4"></a> | <a href="https://www.wikipakk.com/"><img src="https://avatars.githubusercontent.com/u/110786861?s=100&v=4"></a> | <a href="https://www.smapiot.com"><img src="https://avatars.githubusercontent.com/u/44447644?s=100&v=4"></a>

We also had individuals (much appreciated!) that have been very gracious:

- [daveaglick](https://github.com/daveaglick)
- [vmeganathan81](https://github.com/vmeganathan81)
- [jfoshee](https://github.com/jfoshee)
- [Mrxx99](https://github.com/Mrxx99)
- [iamcarbon](https://github.com/iamcarbon)
- [DorYos](https://github.com/DorYos)
- [LukeTOBrien](https://github.com/LukeTOBrien)
- [zunjae](https://github.com/zunjae)
- [Alellujah](https://github.com/Alellujah)
- [Gavin-Williams](https://github.com/Gavin-Williams)

🙏 thanks to everyone!

**Comment by FlorianRappl** [comment]
Also to be clear; AngleSharp and all associated libraries will always be free and MIT licensed.

There is no consequence from no sponsors coming in - except that my time available for the project will definitely be less as compared to having some sponsorship.

One final remark: While GitHub sponsorships are potentially the best way of supporting the project there are also other ways, e.g., getting in touch regarding consulting on AngleSharp best practices if you use it or directly getting in touch regarding development of specific features.

**Comment by FlorianRappl** [comment]
As nothing moves here I am actively thinking about a dual license strategy (one for non-commercial / open-source projects, and another for anything commercial).

Any past contributor or sponsor would get the commercial license for free.

Details are yet to be figured out - I still hope that enough companies support the project to avoid that situation.

**Comment by manutiedra** [comment]
The sponsorship link (https://salt.bountysource.com/teams/anglesharp) does not work

**Comment by FlorianRappl** [comment]
Good catch @manutiedra - seems like bountysource does not longer exist (strange that they never send out an email...). I'll remove the reference. Preferred way is anyway [GitHub Sponsors](https://github.com/sponsors/FlorianRappl).

**Comment by robbendavid** [comment]
FYI, https://github.com/AngleSharp/AngleSharp.Css  shows the broken link

<img width="340" height="166" alt="Image" src="https://github.com/user-attachments/assets/b66e27b7-8700-4dd0-9754-ea33cff1fabf" />
